MerlinAU.sh

```diff
@@ -1012,9 +1044,21 @@ then
     find "$FW_LOG_DIR" -name '*.log' -mtime +30 -exec rm {} \;
 fi
 
-##-------------------------------------##
-## Added by Martinski W. [2024-Jan-24] ##
-##-------------------------------------##
+##---------------------------------------##
+## Added by ExtremeFiretop [2024-Jan-24] ##
+##---------------------------------------##
+if USBMountPoint="$(_GetDefaultUSBMountPoint_)"
```

Comment:
Same as previous comment. The default mount point is not necessarily the one selected by the user. We need to honor the user's selection.